node/src/accountant/payment_adjuster/mod.rs

```diff
@@ -0,0 +1,2340 @@
+use crate::accountant::payment_adjuster::adjustment_runners::{
+    AdjustmentRunner, ServiceFeeOnlyAdjustmentRunner, TransactionAndServiceFeeAdjustmentRunner,
+};
+use crate::accountant::payment_adjuster::criterion_calculators::balance_calculator::BalanceCriterionCalculator;
+use crate::accountant::payment_adjuster::criterion_calculators::CriterionCalculator;
+use crate::accountant::payment_adjuster::logging_and_diagnostics::diagnostics::ordinary_diagnostic_functions::calculated_criterion_and_weight_diagnostics;
+use crate::accountant::payment_adjuster::logging_and_diagnostics::diagnostics::{collection_diagnostics, diagnostics};
+use crate::accountant::payment_adjuster::disqualification_arbiter::{
+    DisqualificationArbiter,
+};
+use crate::accountant::payment_adjuster::inner::{
+    PaymentAdjusterInner, PaymentAdjusterInnerNull, PaymentAdjusterInnerReal,
+};
+use crate::accountant::payment_adjuster::logging_and_diagnostics::log_functions::{
+    accounts_before_and_after_debug,
+};
+use crate::accountant::payment_adjuster::miscellaneous::data_structures::DecidedAccounts::{
+    LowGainingAccountEliminated, SomeAccountsProcessed,
+};
+use crate::accountant::payment_adjuster::miscellaneous::data_structures::{AdjustedAccountBeforeFinalization, AdjustmentIterationResult, RecursionResults, WeightedPayable};
+use crate::accountant::payment_adjuster::miscellaneous::helper_functions::{
+    dump_unaffordable_accounts_by_transaction_fee,
+    exhaust_cw_balance_entirely, find_largest_exceeding_balance,
+    sum_as, zero_affordable_accounts_found,
+};
+use crate::accountant::payment_adjuster::preparatory_analyser::{LateServiceFeeSingleTxErrorFactory, PreparatoryAnalyzer};
+use crate::accountant::payment_adjuster::service_fee_adjuster::{
+    ServiceFeeAdjuster, ServiceFeeAdjusterReal,
+};
+use crate::accountant::scanners::mid_scan_msg_handling::payable_scanner::blockchain_agent::BlockchainAgent;
+use crate::accountant::scanners::mid_scan_msg_handling::payable_scanner::PreparedAdjustment;
+use crate::accountant::{AnalyzedPayableAccount, QualifiedPayableAccount};
+use crate::diagnostics;
+use crate::sub_lib::blockchain_bridge::OutboundPaymentsInstructions;
+use crate::sub_lib::wallet::Wallet;
+use itertools::Either;
+use masq_lib::logger::Logger;
+use std::collections::HashMap;
+use std::fmt::{Display, Formatter};
+use std::time::SystemTime;
+use thousands::Separable;
+use variant_count::VariantCount;
+use web3::types::U256;
+use masq_lib::utils::convert_collection;
+
+// PaymentAdjuster is a very efficient recursive and scalable algorithm that inspects payments under
+// the condition of an acute insolvency. You can expand the scope of the evaluation by writing your
+// own CriterionCalculator, that should be specialized on a distinct parameter of a payable account,
+// and sticking it inside the vector that stores them.
+
+pub type AdjustmentAnalysisResult =
+    Result<Either<Vec<QualifiedPayableAccount>, AdjustmentAnalysis>, PaymentAdjusterError>;
```

Comment:
You wanted to give an alias to `Vec<QualifiedPayableAccount>`. You may use the name `UnadjustedAccounts`.